src/types/OphanTypes.ts

```diff
@@ -0,0 +1,60 @@
+export type OphanProduct =
+    | 'CONTRIBUTION'
+    | 'RECURRING_CONTRIBUTION'
+    | 'MEMBERSHIP_SUPPORTER'
+    | 'MEMBERSHIP_PATRON'
+    | 'MEMBERSHIP_PARTNER'
+    | 'DIGITAL_SUBSCRIPTION'
+    | 'PRINT_SUBSCRIPTION';
```

Comment:
Does GW fall under 'PRINT_SUBSCRIPTION' for the purposes of Ophan?

Reply:
Good question, I'll need to cross check how we're doing it in `frontend`.

Reply:
I had a look and I can't see that any provision was made for differentiating when GW was added (as a banner alternative). I'm guessing it's coming through as 'DIGITAL_SUBSCRIPTION' but I think we should add a new option 'GUARDIAN_WEEKLY'